### PR TITLE
[WIP] Dockerfile for images running on RHEL 7

### PR DESCRIPTION
**Pull request by rootfs**
**Which issue(s) this PR addresses**:

The diff is that
- downgrade from iptables 1.8 to 1.6 (temp workaround, waiting for @mangelajo for final solution)
- remove `metacopy=on` option that is not supported on old kernels

Closes #<Issue Number>

### CODE REVIEW

File: hack/all-in-one/Dockerfile.rhel7 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    sed -i 's|/usr/libexec/crio/conmon|/usr/bin/conmon|' /etc/crio/crio.conf && \
+    systemctl enable microshift.service && \
+    systemctl enable crio 
+
+RUN curl -s -L https://nvidia.github.io/nvidia-docker/rhel8.3/nvidia-docker.repo | tee /etc/yum.repos.d/nvidia-docker.repo && \
+    dnf install nvidia-container-toolkit -y
+
+ADD nvidia-config.toml /etc/nvidia-container-runtime/config.toml
+
+RUN yum remove iptables -y && \
```
Comment (rootfs) [inline hack/all-in-one/Dockerfile.rhel7:37]: @mangelajo help please :D

**Comment by rootfs** [comment]
tested and verified [here](https://gist.github.com/rootfs/bd293ae70aa58751b1a49c143e477505)

File: hack/all-in-one/Dockerfile.rhel7 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    yum install -y https://rpmfind.net/linux/opensuse/distribution/leap/15.1/repo/oss/x86_64/xtables-plugins-1.6.2-lp151.2.3.x86_64.rpm && \
+    yum install -y https://rpmfind.net/linux/opensuse/distribution/leap/15.1/repo/oss/x86_64/iptables-1.6.2-lp151.2.3.x86_64.rpm
```
Comment (mangelajo) [inline hack/all-in-one/Dockerfile.rhel7:39]: We need to find stuff from fedora at least. I'll give it a spin this morning and try to figure out a better way .

**Comment by rootfs** [comment]
will reopen from my own repo branch

**Pull request closed without merging**